workflows_api/runtime/src/collection_publisher.py

```diff
@@ -58,21 +58,25 @@ class Publisher:
 
     def __init__(self) -> None:
         self.func_map = {
-            DataType.zarr: self.create_zarr_collection,
-            DataType.cog: self.create_cog_collection,
+            DataType.zarr.value: self.create_zarr_collection,
+            DataType.cog.value: self.create_cog_collection,
```

Comment:
🤔

Reply:
the expected map is between string key and function value. It was rather <Datatype.cog> as key and function as value.